test/Confluent.Kafka.IntegrationTests/Tests/AdminClient_DeleteConsumerGroupOffsets.cs

```diff
@@ -0,0 +1,101 @@
+
+using System;
+using System.Collections.Generic;
+using Xunit;
+
+
+namespace Confluent.Kafka.IntegrationTests
+{
+    public partial class Tests
+    {
+        /// <summary>
+        ///     Test functionality of AdminClient.DeleteConsumerGroupOffsets.
+        /// </summary>
+        [Theory, MemberData(nameof(KafkaParameters))]
+        public void AdminClient_DeleteConsumerGroupOffsets(string bootstrapServers)
+        {
+            LogToFile("start AdminClient_DeleteConsumerGroupOffsets");
+
+            using (var topic1 = new TemporaryTopic(bootstrapServers, 1))
+            using (var topic2 = new TemporaryTopic(bootstrapServers, 1))
+            using (var adminClient = new AdminClientBuilder(new AdminClientConfig { BootstrapServers = bootstrapServers }).Build())
+            using (var consumer = new ConsumerBuilder<Null, string>(new ConsumerConfig { BootstrapServers = bootstrapServers, GroupId = topic1.Name }).Build())
+            {
+                var groupId = topic1.Name;
+                var offsetToCommit = 10;
+
+                consumer.Assign(new List<TopicPartition>(){new TopicPartition(topic1.Name, 0)});
+                consumer.Commit(new List<TopicPartitionOffset>() {new TopicPartitionOffset(topic1.Name, 0, offsetToCommit)}); //commit some offset for consumer
+
+                var committedOffsets = consumer.Committed(TimeSpan.FromSeconds(10));
+                Assert.Equal(1, committedOffsets.Count);
+                Assert.Equal(offsetToCommit, committedOffsets[0].Offset);
+
+                List<TopicPartitionOffset> topicPartitionToReset = new List<TopicPartitionOffset>() {new TopicPartitionOffset(topic1.Name, 0, -1)}; //offset value is ignored
+
+                var res = adminClient.DeleteConsumerGroupOffsetsAsync(groupId, topicPartitionToReset).Result;
+                Assert.Equal(1, res.Count);
+                Assert.Equal(groupId, res[0].Group);
+                Assert.Equal(1, res[0].Partitions.Count);
+                Assert.Equal(0, res[0].Partitions[0].Partition.Value);
+
+                committedOffsets = consumer.Committed(TimeSpan.FromSeconds(1));
+                Assert.Equal(1, committedOffsets.Count);
+                Assert.Equal(-1001, committedOffsets[0].Offset); //offset is unset
+
+                // Consumer is actively subscribed to the topic
+                consumer.Subscribe(new List<String>(){topic1.Name});
+                consumer.Commit(new List<TopicPartitionOffset>() {new TopicPartitionOffset(topic1.Name, 0, offsetToCommit)}); //commit some offset for consumer
+
+                committedOffsets = consumer.Committed(TimeSpan.FromSeconds(10));
+                Assert.Equal(1, committedOffsets.Count);
+                Assert.Equal(offsetToCommit, committedOffsets[0].Offset);
+
+                topicPartitionToReset = new List<TopicPartitionOffset>() {new TopicPartitionOffset(topic1.Name, 0, -1)}; //offset value is ignored
+                Assert.Equal(1, res.Count);
```

Comment:
you are not calling `DeleteConsumerGroupOffsetsAsync` here, so of course the committed offset is not deleted.

Reply:
Yep, somehow missed this. Updated now.